src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/config/BundledNotificationsConfigTestIt.java

```diff
@@ -0,0 +1,71 @@
+import static uk.gov.hmcts.reform.sscs.domain.notify.NotificationEventType.ACTION_POSTPONEMENT_REQUEST;
+import static uk.gov.hmcts.reform.sscs.domain.notify.NotificationEventType.ACTION_POSTPONEMENT_REQUEST_WELSH;
+import static uk.gov.hmcts.reform.sscs.domain.notify.NotificationEventType.DECISION_ISSUED;
+import static uk.gov.hmcts.reform.sscs.domain.notify.NotificationEventType.DECISION_ISSUED_WELSH;
+import static uk.gov.hmcts.reform.sscs.domain.notify.NotificationEventType.ISSUE_ADJOURNMENT_NOTICE;
+import static uk.gov.hmcts.reform.sscs.domain.notify.NotificationEventType.ISSUE_ADJOURNMENT_NOTICE_WELSH;
+import static uk.gov.hmcts.reform.sscs.domain.notify.NotificationEventType.ISSUE_FINAL_DECISION;
+import static uk.gov.hmcts.reform.sscs.domain.notify.NotificationEventType.ISSUE_FINAL_DECISION_WELSH;
+
+import java.util.ArrayList;
+import java.util.EnumSet;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import junitparams.Parameters;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+import uk.gov.hmcts.reform.sscs.domain.notify.NotificationEventType;
+import uk.gov.hmcts.reform.sscs.domain.notify.Template;
+
+public class BundledNotificationsConfigTestIt extends AbstractNotificationConfigTest {
+    private static final Set<NotificationEventType> BUNDLED_LETTER_EVENT_TYPES_IGNORED = EnumSet.of(
+        ACTION_POSTPONEMENT_REQUEST,
+        ACTION_POSTPONEMENT_REQUEST_WELSH,
+        DECISION_ISSUED,
+        DECISION_ISSUED_WELSH,
+        ISSUE_FINAL_DECISION,
+        ISSUE_FINAL_DECISION_WELSH,
+        ISSUE_ADJOURNMENT_NOTICE,
+        ISSUE_ADJOURNMENT_NOTICE_WELSH
+    );
+    private static final Set<AppealHearingType> APPEAL_HEARING_TYPES = Set.of(PAPER, ORAL);
+
+    @Test
+    @Parameters(method = "bundledLetterTemplateNames")
+    public void given_bundledLetters_should_notHaveTemplate(NotificationEventType eventType) {
+        List<Template> templates = getTemplates(eventType);
+
+        assertThat(templates).allSatisfy(template -> {
+            assertThat(template.getEmailTemplateId()).isNull();
+            assertThat(template.getSmsTemplateId()).isEmpty();
+            assertThat(template.getLetterTemplateId()).isNull();
+            assertThat(template.getDocmosisTemplateId()).isNull();
+        });
+    }
+
+    @NotNull
+    private List<Template> getTemplates(NotificationEventType eventType) {
+        List<Template> templates = new ArrayList<>();
+        for (SubscriptionType subscriptionType : SubscriptionType.values()) {
+            for (AppealHearingType appealHearingType : APPEAL_HEARING_TYPES) {
+                templates.add(getTemplate(eventType, subscriptionType, appealHearingType, true, null));
+                templates.add(getTemplate(eventType, subscriptionType, appealHearingType, false, null));
+            }
+        }
+        return templates;
+    }
+
+    @SuppressWarnings({"unused"})
```

Comment:
Why is this `@SuppressWarning` necessary?

Reply:
Its an annoyance with junit 4's parameter runner, I would really like to see all repos upgraded to jypter 5 tbh.
Without it causes a sonarLint code smell:
![image](https://user-images.githubusercontent.com/18538046/193086510-61ebf2e9-8a64-40f0-b31f-15c6748d59c9.png)

Reply:
I see. Makes sense.